lib/generators/sufia/templates/migrations/create_user_stats.rb

```diff
@@ -5,15 +5,18 @@ def change
       t.datetime :date
       t.integer :file_views
       t.integer :file_downloads
+      t.integer :work_views
```

Comment:
We'll want this in a new migration that your new generator (re: :arrow_heading_up:) can include.